src/core/css/ErrorHandler.scss

```diff
@@ -7,4 +7,8 @@
   padding: 10px;
   margin: 0;
   margin-bottom: 10px;
+
+  > li {
```

Comment:
Trying to keep AMO consistent is a bit dangerous since everything is in flux but that's a fair point about selector specificity. I switched it to a class.